src/status_im2/contexts/activity_center/style.cljs

```diff
@@ -60,3 +53,8 @@
    :height           120
    :background-color colors/danger-50
    :margin-bottom    20})
+
+(def blur
+  {:style       {:position :absolute :top 0 :left 0 :right 0 :bottom 0}
+   :blur-amount 20
+   :blur-type   :dark})
```

Comment:
In the designs, the background color is `colors/neutral-80-opa-80`. I think this color combined with `:blur-amount 5` reflect the designs more accurately (without blur-type).